ui/pages/Project/components/FamilyPage.jsx

```diff
@@ -201,16 +203,59 @@ const mapStateToProps = (state, ownProps) => ({
 
 export const FamilyDetail = connect(mapStateToProps)(BaseFamilyDetail)
 
-const FamilyPage = ({ match }) => (
+const MyFamilyPage = ({ match }) => (
   <FamilyDetail
-    familyGuid={match.params.familyGuid}
+    familyGuid={match.url.split('/')[4]}
```

Comment:
You need to pass an explicit `familyGuid` from the router to the routes, same as we do with the `projectGuid` to all the project pages

Reply:
The project pages use the `currentProjectGuid` in the state. Do we have to do the same thing?